src/components/common.js

```diff
@@ -528,10 +528,12 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) =
         `${getAppName(true)} is a ${Utils.cond(
           [isTerra(), () => 'cloud-native platform'],
           [isFirecloud(), () => 'NCI Cloud Resource project powered by Terra'],
+          [isProjectSingular(), () => 'project funded by Additional Ventures and powered by Terra'],
           () => 'project powered by Terra'
         )} for biomedical researchers to `,
         heavyWrapper('access data'), ', ', heavyWrapper('run analysis tools'), ', ',
-        span({ style: { whiteSpace: 'nowrap' } }, ['and', heavyWrapper(' collaborate'), '.'])
+        h(Fragment, [span({ style: { whiteSpace: 'nowrap' } }, ['and', strong([' collaborate'])]),
+          isProjectSingular() && ' to advance research around single ventricle heart disease', '.'])
```

Comment:
```suggestion
        span({ style: { whiteSpace: 'nowrap' } }, ['and', strong([' collaborate'])]),
        isProjectSingular() && ' to advance research around single ventricle heart disease', '.'
```

Now that this is inlined, I don't think the Fragment is necessary.

Reply:
Thank you! Addressed in [d3bf3a7](https://github.com/DataBiosphere/terra-ui/pull/3134/commits/d3bf3a7fe23dc8c5c801f2a394f4424a52764570).